benchmark-scripts/igt-entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+devices=$(/usr/local/bin/intel_gpu_top -L  | grep card | awk '{print $1,$5;}' | sed ":a;N;s/\n/@/g")
+echo "devices found $devices"
```

Comment:
check devices env if empty or not found devices and return different error code in that case

Reply:
Added